department-of-veterans-affairs/caseflow
# Call out sort direction on table sort icons

Resolves #12136

### Description
Update aria labels to include sort direction when sorting a column

### Acceptance Criteria
- [x] Works in reader table view
- [x] Works in queue table view
- [x] When the column is not being sorted, it does not announce a sort direction
- [x] When the column is sorted ascending, "Sorted ascending" is read
- [x] When the column is sorted descending, "Sorted descending" is read

### Testing Plan
1. Log in as bvaaabshire and go to your queue
1. With voice over on, tab to the sort icon on the queue table.
   - [x] When the column is not being sorted, announce no sort direction
1. Click the sort icon and tab back to it
   - [x] When the column is sorted ascending, "Sorted ascending" is read
1. Click the sort icon again and tab back to it
   - [x] When the column is sorted descending, "Sorted descending" is read
1. Click into case details and click into the veteran documents
1. Repeat the above with the document list (note that the doc list is sorted by receipt date by default)

### User Facing Changes
 - [x] Sort order is read in queue and document tables
